executor/show.go

```diff
@@ -624,3 +628,28 @@ func (e *ShowExec) getTable() (table.Table, error) {
 	}
 	return tb, nil
 }
+
+func (e *ShowExec) fetchShowStatsMeta() error {
```

Comment:
Because the memory may not store the newest information. At least you should trigger the `Update` action.